polkadot-parachains/src/command.rs

```diff
@@ -161,6 +169,14 @@ fn use_shell_runtime(chain_spec: &dyn ChainSpec) -> bool {
 	chain_spec.id().starts_with("shell")
 }
 
+fn use_statemine_runtime(chain_spec: &dyn ChainSpec) -> bool {
+	chain_spec.id().starts_with("statemine")
+}
+
+fn use_westmint_runtime(chain_spec: &dyn ChainSpec) -> bool {
+	chain_spec.id().starts_with("westmint")
+}
+
 use crate::service::{new_partial, RococoParachainRuntimeExecutor, ShellRuntimeExecutor};
```

Comment:
Do the other runtimes executors need to be added?
`StatemintRuntimeExecutor`, `StatemineRuntimeExecutor`, `WestmintRuntimeExecutor`

Reply:
https://github.com/paritytech/statemint/blob/9e8d640dfe0311652d848bfd0aea3ae62bc54b0f/node/src/service.rs#L41
https://github.com/paritytech/statemint/blob/9e8d640dfe0311652d848bfd0aea3ae62bc54b0f/node/src/service.rs#L49
https://github.com/paritytech/statemint/blob/9e8d640dfe0311652d848bfd0aea3ae62bc54b0f/node/src/service.rs#L57

Reply:
Done.